src/libraries/Microsoft.Extensions.Hosting.WindowsServices/src/WindowsServiceLifetime.cs

```diff
@@ -47,7 +47,7 @@ public Task WaitForStartAsync(CancellationToken cancellationToken)
             cancellationToken.Register(() => _delayStart.TrySetCanceled());
             ApplicationLifetime.ApplicationStarted.Register(() =>
             {
-                Logger.LogInformation("Application started. Hosting environment: {envName}; Content root path: {contentRoot}",
+                Logger.LogInformation("Application started. Hosting environment: {EnvName}; Content root path: {ContentRoot}",
```

Comment:
Might this be a breaking change? e.g. are these placeholder strings passed along to the ILogger in a way where it might be comparing them?